docs/fidesops/docs/guides/reporting.md

```diff
@@ -51,6 +51,8 @@ If an `external_id` was provided at request creation, we can also track the priv
 
 `GET api/v1/privacy-request?external_id=<external_id>`
 
+NB. These parameters will return matching Privacy Requests based on startswith matches.
```

Comment:
What does `NB.` mean here?

Reply:
Apologies, was under the impression this was common vernacular:

![Screenshot 2022-04-05 at 13 43 55](https://user-images.githubusercontent.com/1667340/161817568-55e43b19-94a3-4678-a092-bebddfb47782.png)

Can change this to `Please note:` or similar